web/modules/custom/do_base/do_base.deploy.php

```
@@ -8,3 +8,225 @@
+  });
+}
+
+/**
+ * Rebuild the homepage (node 1) to the redesign.
+ *
+ * Sets the hero (banner) copy and replaces the page components with the
+ * redesign sections, each rendered as a full-width dark content paragraph from
+ * the markup in this module's content/homepage directory.
+ */
+function do_base_deploy_homepage(): string {
+  $node = Node::load(1);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Drupal\node\Entity\Node' in method 'do_base_deploy_homepage'. (undefined)

(StaticAccess)